src/lib/core/a11y/interactivity-checker.ts

```diff
@@ -30,26 +33,93 @@ export class InteractivityChecker {
    * being clipped by an `overflow: hidden` parent or being outside the viewport.
    */
   isVisible(element: HTMLElement) {
-    // There are additional special cases that this does not capture, but this will work for
-    // the most common cases.
+    let nodeName = element.nodeName.toLowerCase();
+    let isControlAudio = nodeName === 'audio' && (element as HTMLAudioElement).controls;
 
-    // Use logic from jQuery to check for `display: none`.
-    // See https://github.com/jquery/jquery/blob/master/src/css/hiddenVisibleSelectors.js#L12
-    if (!(element.offsetWidth || element.offsetHeight || element.getClientRects().length)) {
+    // In IE11 audio elements with controls have invalid rectangles, but are still visible.
+    if (!isControlAudio && !checkRectangles()) {
```

Comment:
an `<audio>` with `visibility: hidden` should be fine, but I think it does miss the `display: none` case, maybe special-case `TRIDENT`/`EDGE` since other browsers have the correct behavior anyways